src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AppDesigner.xaml

```diff
@@ -1,27 +1,37 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
-<Rule
-    Name="AppDesigner"
-    PageTemplate="generic"
-    xmlns="http://schemas.microsoft.com/build/2009/properties">
-
+<Rule Name="AppDesigner"
+      PageTemplate="generic"
+       xmlns="http://schemas.microsoft.com/build/2009/properties">
```

Comment:
There's an extra space here that's going to get removed on reformat.

Reply:
Fixed and reformatted all